app/desmos/cmd/init.go

```diff
@@ -0,0 +1,32 @@
+package cmd
+
+// DONTCOVER
+
+import (
+	"path/filepath"
+
+	"github.com/spf13/cobra"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	srvconfig "github.com/cosmos/cosmos-sdk/server/config"
+	"github.com/cosmos/cosmos-sdk/types/module"
+	cosmosgenutilcli "github.com/cosmos/cosmos-sdk/x/genutil/client/cli"
+)
+
+// initCmd returns a command that initializes all files needed for Tendermint
+// and the respective application.
+func initCmd(mbm module.BasicManager, defaultNodeHome string) *cobra.Command {
+	cmd := cosmosgenutilcli.InitCmd(mbm, defaultNodeHome)
+	defaultRun := cmd.RunE
+	cmd.RunE = func(cmd *cobra.Command, args []string) error {
+		clientCtx := client.GetClientContextFromCmd(cmd)
+		err := defaultRun(cmd, args)
```

Comment:
We should check for errors here
```suggestion
		err := defaultRun(cmd, args)
		if err != nil {
			return err
		}
```